### PR TITLE
feat(hbase): install the hostname command

### PR DESCRIPTION
**Pull request by razvan**
# Description

part of https://github.com/stackabletech/hbase-operator/issues/508

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Integration tests ran successfully
```

<details>
<summary>TIP: Running integration tests with a new product image</summary>

The image can be built and uploaded to the kind cluster with the following commands:

```shell
bake --product <product> --image-version <stackable-image-version>
kind load docker-image <image-tagged-with-the-major-version> --name=<name-of-your-test-cluster>
```

See the output of `bake` to retrieve the image tag for `<image-tagged-with-the-major-version>`.
</details>

### CODE REVIEW

**Review by sbernauer — APPROVED** [review]
Please wait with merge until just before merging the operator PR. There might be the case we don't need it

File: hbase/Dockerfile [hunk 1]
```diff
@@ -332,7 +332,11 @@ microdnf update
 
 # The tar and python packages are required by the Phoenix command line.
 # We add zip and gzip because tar without compression is seldom useful.
+# TODO: maybe hostname is useful enough to have in stackable-base ?
+# hostname is required by the hbase/bin/graceful_stop.sh to identify the actual name
+# of the region server that is being shut down.
```
Comment (NickLarsenNZ) [inline hbase/Dockerfile:337]: I think it is useful enough to put in stackable-base.

**Review by NickLarsenNZ — APPROVED** [review]
LGTM, left a comment

**Comment by razvan** [comment]
obsolete. SDP doesn't use the graceful shutdown script.

**Pull request closed without merging**